docs-src/0.4/en/reference/router.md

```diff
@@ -87,4 +87,4 @@ rsx!{
 
 ## More reading
 
-This page is just a very brief overview of the router. For more information, check out [the router book](../../router/index.md) or some of [the router examples](https://github.com/DioxusLabs/dioxus/blob/master/examples/router.rs).
+This page is just a very brief overview of the router. For more information, check out [the router reference](../router/index.md) or some of [the router examples](https://github.com/DioxusLabs/dioxus/blob/master/examples/router.rs).
```

Comment:
The router reference is a different link (`https://dioxuslabs.com/learn/0.4/router/reference`). This link should go to the router book which explains the difference between the router reference and the router guide

Reply:
I'm not sure what you mean by "book". This edit links to [learn/0.4/router](https://dioxuslabs.com/learn/0.4/router), which is what I thought is the book, but it's not given that title anywhere. That's why I renamed it to the reference. Previously, the link was broken.

Reply:
> This edit links to [learn/0.4/router](https://dioxuslabs.com/learn/0.4/router), which is what I thought is the book, but it's not given that title anywhere.

That is the section I was talking about as the router book (or just the router docs).

The reference is a [section under the router book](https://dioxuslabs.com/learn/0.4/router/reference). Calling both the larger router book and the section under the book the "router reference" could get confusing.

Reply:
Well I got confused by a "router book" link when there is no explicit "router book" :) Maybe it would be best to rename each section/book for a specific tool (currently Router and CLI) to "Router book" and "CLI book".

To be honest, I think "Reference > Routing" is unnecessary, because the Router book goes over basically the exact same thing. I think some of the other "Reference > X" sections should be moved to their own respective book, like each renderer.

Reply:
How about `router section`, `router docs` or `[router](link) book`? As long as it is not the name of a different page, it probably works.

Adding book to each section in the nav bar seems a bit redundant to me.

> To be honest, I think "Reference > Routing" is unnecessary, because the Router book goes over basically the exact same thing. I think some of the other "Reference > X" sections should be moved to their own respective book, like each renderer.

Feel free to open another issue to discuss reorganizing things

Reply:
> Adding book to each section in the nav bar seems a bit redundant to me

I guess. Let's just keep it consistent with calling it "book" then.